bandit.yaml

```diff
@@ -0,0 +1,2 @@
+skips: ['B303']
```

Comment:
That's going to disable the check throughout the entire code, isn't it?  I don't think that's appropriate.  Even if all current violations of the rule are not a problem, what if someone introduced new code which violates this rule and it really is a problem?  It wouldn't be detected.

I think it needs to be handled instead as described in: https://bandit.readthedocs.io/en/latest/config.html#suppressing-individual-lines